src/components/plots/xyPlot.ts

```diff
@@ -181,24 +176,6 @@ module Plottable {
       return this;
     }
 
-    protected _updateXDomainer() {
-      if (this.x().scale instanceof QuantitativeScale) {
-        var scale = <QuantitativeScale<any>> this.x().scale;
-        if (!scale._userSetDomainer) {
-          scale.domainer().pad().nice();
-        }
-      }
-    }
-
-    protected _updateYDomainer() {
-      if (this.y().scale instanceof QuantitativeScale) {
-        var scale = <QuantitativeScale<any>> this.y().scale;
-        if (!scale._userSetDomainer) {
-          scale.domainer().pad().nice();
-        }
-      }
-    }
-
```

Comment:
`Scale`s now pad by default, and `nice()`ing is included in the padding. I'll make a note of it in the PR description.